pkg/actors/actors.go

```diff
@@ -86,86 +91,38 @@ type Actors interface {
-	activeReminders       *sync.Map
-	remindersLock         sync.RWMutex
-	remindersStoringLock  sync.Mutex
-	reminders             map[string][]actorReminderReference
-	evaluationLock        sync.RWMutex
-	evaluationChan        chan struct{}
-	appHealthy            *atomic.Bool
-	certChain             *daprCredentials.CertChain
-	tracingSpec           configuration.TracingSpec
-	resiliency            resiliency.Provider
-	storeName             string
-	compStore             *compstore.ComponentStore
-	ctx                   context.Context
-	cancel                context.CancelFunc
-	clock                 clock.WithTicker
-	internalActors        map[string]InternalActor
-	internalActorChannel  *internalActorChannel
+	appChannel           channel.AppChannel
+	placement            internal.PlacementService
+	grpcConnectionFn     GRPCConnectionFn
+	actorsConfig         Config
+	timers               internal.TimersProvider
+	actorsReminders      internal.RemindersProvider
+	actorsTable          *sync.Map
+	appHealthy           *atomic.Bool
+	certChain            *daprCredentials.CertChain
+	tracingSpec          configuration.TracingSpec
+	resiliency           resiliency.Provider
+	storeName            string
+	compStore            *compstore.ComponentStore
+	ctx                  context.Context
+	cancel               context.CancelFunc
+	clock                clock.WithTicker
+	internalActors       map[string]InternalActor
+	internalActorChannel *internalActorChannel
 
 	// TODO: @joshvanl Remove in Dapr 1.12 when ActorStateTTL is finalized.
 	stateTTLEnabled bool
 }
 
-// ActorMetadata represents information about the actor type.
-type ActorMetadata struct {
-	ID                string                 `json:"id"`
-	RemindersMetadata ActorRemindersMetadata `json:"actorRemindersMetadata"`
-	Etag              *string                `json:"-"`
-}
-
-// ActorRemindersMetadata represents information about actor's reminders.
-type ActorRemindersMetadata struct {
-	PartitionCount int                `json:"partitionCount"`
-	partitionsEtag map[uint32]*string `json:"-"`
-}
-
-type actorReminderReference struct {
-	actorMetadataID           string
-	actorRemindersPartitionID uint32
-	reminder                  reminders.Reminder
-}
-
-var (
-	ErrIncompatibleStateStore   = errors.New("actor state store does not exist, or does not support transactions which are required to save state - please see https://docs.dapr.io/operations/components/setup-state-store/supported-state-stores/")
-	ErrDaprResponseHeader       = errors.New("error indicated via actor header response")
-	ErrReminderCanceled         = errors.New("reminder has been canceled")
-	ErrReminderOpActorNotHosted = errors.New("operations on actor reminders are only possible on hosted actor types")
-)
-
 // ActorsOpts contains options for NewActors.
 type ActorsOpts struct {
 	AppChannel       channel.AppChannel
 	GRPCConnectionFn GRPCConnectionFn
-	Config           Config
+	ActorsConfig     Config
```

Comment:
I think keeping this `Config` rather than `ActorsConfig` since `Actors` is implied by the package and  type.

Reply:
Done